src/main/java/org/mockito/internal/stubbing/defaultanswers/ForwardsInvocations.java

```diff
@@ -40,6 +40,11 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
             }
 
             Object[] rawArguments = ((Invocation) invocation).getRawArguments();
+            try {
+                delegateMethod.setAccessible(true);
```

Comment:
Can you add an e2e user test? Can you put together a test class that shows that without this change Mockito indeed throws access exceptions?